src/core/graphics/webgl/shaders/PrimitiveShader.js

```diff
@@ -29,18 +31,16 @@ class PrimitiveShader extends Shader
                 'void main(void){',
                 '   gl_Position = vec4((projectionMatrix * translationMatrix * vec3(aVertexPosition, 1.0)).xy, 0.0, 1.0);',
                 '   vColor = aColor * vec4(tint * alpha, alpha);',
-                '}'
+                '}',
```

Comment:
Cleans up diffs. If you add/change a line it doesn't have changes elsewhere in the diff. By always having a trailing comma you ensure your diffs remain very clean and obvious as to what has changed. Then we can let the transpiler deal with removing it if necessary for compat.